README.md

```diff
@@ -144,6 +145,36 @@ $ spanner-cli -p myproject -i myinstance -d mydb -e 'SELECT * FROM users;' -t
 +----+------+--------+
 ```
 
+### Directed reads mode
+
+spanner-cli now supports Directed reads, a feature that allows you to read data from a specific replica of a Spanner database. 
+To use Directed reads with spanner-cli, you need to specify the `--directedread` flag.
+The `--directedread` flag takes a single argument, which is the name of the replica that you want to read from.
+The replica name can be specified in one of the following formats:
+
+- `<location>`
```

Comment:
Let's use consistent naming over document and code. Currently code throws error with `<replica_location>` and `<replica_type>`.

I think either is fine, but consistent is important: `<location>:<type>` or `<replica_location>:<replica_type>`.

Reply:
I've unified the inconsistent.